### ISSUE TITLE
REST GET call with compound primary key does not work in Swagger UI

### ISSUE THREAD

**Issue by jdavies** [stargate/stargate#1004]
I'm trying to use the REST API to do.a GET /v2/keyspaces/{keyspaceName}/{tableName}/{primaryKey} where the primary key contains 2 fields. When I test this in Swagger it fails because the "/" character between the primary key fields is URL encoded as %2C. For example, this is the CURL that is generated and executed in Swagger:

```curl -X GET "https://dad43061-23d9-43ec-9c1f-6de1f7349a61-us-west-2.apps.astra.datastax.com/api/rest/v2/keyspaces/kcorp/login/jeff.davies%40datastax.com%2C05ccb21ac9e1f27aa25dddbccc5251cb010ea99a949768212665e7a6151fdf64?fields=id" -H  "accept: application/json" -H  "X-Cassandra-Token: AstraCS:<REDACTED>"```

But this call works after I've edited the URL:

```curl -X GET "https://dad43061-23d9-43ec-9c1f-6de1f7349a61-us-west-2.apps.astra.datastax.com/api/rest/v2/keyspaces/kcorp/login/jeff.davies%40datastax.com/05ccb21ac9e1f27aa25dddbccc5251cb010ea99a949768212665e7a6151fdf64?fields=id" -H  "accept: application/json" -H  "X-Cassandra-Token: AstraCS:<REDACTED>"```

The result is that the Swagger API cannot be used to test the GET call.

**Comment by dimas-b** [comment]
HI @jdavies : Is it just the `curl` example in Swagger perhaps? Can Swagger execute the requests in its UI?

**Comment by dougwettlaufer** [comment]
`%2C` is actually `,`. It looks like swagger is concatenating the keys with comma since primary key is modeled as a list.

Have you tried putting them all in one text field with the `/` delimiter?
<img width="680" alt="Screen Shot 2021-06-09 at 4 00 38 PM" src="https://user-images.githubusercontent.com/45750136/121428898-f2482580-c93b-11eb-80f2-b72fe6576b09.png">

**Comment by jdavies** [comment]
Yes, I have tried it by using the slash with both data fields in a single swagger primary key text field. The command also fails in the Swagger UI, not just using the curl command line. It returns an empty result. I have attached screenshots. I just generated these from Astra now.
<img width="991" alt="Screen Shot 2021-06-10 at 7 44 52 AM" src="https://user-images.githubusercontent.com/297151/121546076-e1d98e80-c9bf-11eb-9eaf-91abfaeda13f.png">
<img width="1420" alt="Screen Shot 2021-06-10 at 7 45 15 AM" src="https://user-images.githubusercontent.com/297151/121546095-e4d47f00-c9bf-11eb-8a08-cbdbca2e395a.png">
